package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Birthday App Reminder",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest --watch",
```

Comment:
Careful with this.

This command (`jest --watch`) won't exit, so your CI won't like it.